### DISCUSSION TITLE
Show related panels in the datasource edit page.

### DISCUSSION THREAD

**Discussion by kevisong** [grafana/grafana#47096] [Data Sources]
Once a datasource is deleted. multiple panels using the datasource maybe affected. 

When there are many dashboards, it will be better if we can see related panels in the datasource edit page, just to know home many panels will be affected if the datasource is edited or deleted.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
